manoseimas/mps_v2/models.py

```diff
@@ -113,3 +113,32 @@ def __unicode__(self):
         return u'{} - {} ({})'.format(self.group.name,
                                       self.member.full_name,
                                       self.position)
+
+
+class Stenogram(CrawledItem):
+    source_id = models.CharField(max_length=16, db_index=True)
+    date = models.DateField()
+    sitting_no = models.IntegerField()
+    sitting_name = models.TextField(blank=True, null=True)
+
+    def __unicode__(self):
+        return u'{} Nr. {}'.format(self.date, self.sitting_no)
+
+
+class StenogramTopic(CrawledItem):
+    stenogram = models.ForeignKey(Stenogram, related_name='topics')
+    title = models.TextField()
```

Comment:
If I would have datetime here (when this topic was started), it would let me identify right question with higher precision, now I have to rely only on document titles.